src/pages/pvNewsletter.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
Same thing here, you could just reuse the original Newsletter component rather than rewriting the code.